src/serac/infrastructure/input.cpp

```diff
@@ -24,8 +24,11 @@ axom::inlet::Inlet initialize(axom::sidre::DataStore& datastore, const std::stri
   luareader->parseFile(input_file_path);
 
   // Store inlet data under its own group
+  if (datastore.getRoot()->hasGroup("input_file")) {
+    // If this is a restart, wipe out the previous input file
+    datastore.getRoot()->destroyGroup("input_file");
```

Comment:
If I'm understanding this correctly, Inlet doesn't have the capability to reconstruct its internal hierarchy from the corresponding Sidre data (so we'd need a "fresh" input file for everything).  I imagine this would be a useful feature for Inlet, though, since restarts were one of the primary motivations for using Sidre for data storage.

Reply:
Yeah, I think this is fine for now but we should document this restriction and log a feature request in Inlet.

Reply:
Logged as https://github.com/LLNL/axom/issues/487

Reply:
Should we save the path of the input file in sidre and load from that so the user doesn't have to specify?